.github/workflows/buildcheck.yml

```diff
@@ -30,7 +30,25 @@ jobs:
         run: |
           pytest -v
 
+      - name: Prepare for test using latest VSC
+        run: |
+          git clone https://github.com/COVESA/vehicle_service_catalog/
+
       - name: Run example tool on latest VSC
         run: |
-          git clone https://github.com/COVESA/vehicle_service_catalog/ -b input_output
```

Comment:
No need to use branch "input_output" any longer as https://github.com/COVESA/vehicle_service_catalog/pull/42 has been merged